pants_plugins/sendwave/pants_node/subsystems.py

```diff
@@ -24,24 +25,17 @@ class NodeSubsystem(Subsystem):
     options_scope = "node"
     help = "Node Options."
 
-    @classmethod
-    def register_options(cls, register):
-        """Register Sendwave pants-node options."""
-        super().register_options(register)
-        register(
-            "--search-paths",
-            type=list,
-            member_type=str,
-            default=["/bin", "/usr/bin/"],
-            help="Directories in which to search for node binaries.'",
-        )
-        register(
-            "--use-nvm",
-            type=bool,
-            default=True,
-            help="If true, the value of $NVM_BIN will be "
-            "appended to the front of the search path.",
-        )
+    search_paths = StrListOption(
+        "--search-paths",
+        default=["/bin", "/usr/bin/"],
+        help="Directories in which to search for node binaries.'",
+    )
+    use_nvm = BoolOption(
+        "--use-nvm",
+        default=True,
+        help="If true, the value of $NVM_BIN will be "
+             "appended to the front of the search path.",
+    )
```

Comment:
`Subsystem.register_options` is now deprecated in favor of the more declarative `Option` syntax: https://github.com/pantsbuild/pants/blob/main/src/python/pants/notes/2.11.x.md#plugin-api-changes-1